src/icons/fileIcons.ts

```diff
@@ -1473,5 +1473,15 @@ export const fileIcons: FileIcons = {
       fileNames: ['jsconfig.json'],
       fileExtensions: ['jsconfig.json'],
     },
+    {
+      name: 'ember',
+      fileNames: [
+          '.ember-cli',
+          '.ember-cli.js',
+          'ember-cli-builds.js',
+          'fastboot.js',
```

Comment:
Is `fastboot.js` a config file or something like that? If it's just the library itself I'd recommend to use the JavaScript icon instead. For example the same pattern applies for files like `jquery.js` or `boostrap.css` files. They also do **not show** the logo of the libraries but simply the icon of the language like CSS or JS.

Reply:
I thought I removed Fastboot, it is a config file but it also conflicts with the Android and Angular fastboot.js files..